clash-lib/src/Clash/Netlist/Types.hs

```diff
@@ -498,6 +499,10 @@ data PortMap
   --
   deriving (Show)
 
+-- | @component@ declaration
```

Comment:
Can you add more documentation? Probably refer to relevant VHDL sources.

Reply:
Yes!